.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -31,22 +31,18 @@ A clear and concise description of what you expected to happen.
 If applicable, add screenshots to help explain your problem.
 
 **Application details**
-* Version of FreeRDP
+* FreeRDP version (xfreerdp /version)
 * Command line used
-* output of `/buildconfig`
-* OS version connecting to
+* Output of `xfreerdp /buildconfig`
+* OS version connecting to (server side)
 * If available the log output from a run with `/log-level:trace` 
+* If you built it yourself add some notes which tag/commit/branch you have used, also your cmake parameters and
```

Comment:
Maybe add `/log-level:trace 2>&1 |tee log.txt` or similar example as howto (had to answer that more than once)